app/src/pages/instance/projectsPage/modals/addProjectModal.jsx

```diff
@@ -14,97 +14,76 @@
  * limitations under the License.
  */
 
-import React, { Component } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
-import { injectIntl } from 'react-intl';
+import { useIntl } from 'react-intl';
 import { reduxForm } from 'redux-form';
 import { FieldErrorHint } from 'components/fields/fieldErrorHint';
 import { FieldProvider } from 'components/fields/fieldProvider';
-import { Input } from 'components/inputs/input';
-import { validate, bindMessageToValidator, validateAsync } from 'common/utils/validation';
-import { ModalLayout, withModal, ModalField } from 'components/main/modal';
-import { SectionHeader } from 'components/main/sectionHeader';
+import { commonValidators } from 'common/utils/validation';
+import { withModal } from 'components/main/modal';
 import { COMMON_LOCALE_KEYS } from 'common/constants/localization';
+import { Modal } from '@reportportal/ui-kit';
+import { FieldText } from 'componentLibrary/fieldText';
+import { hideModalAction } from 'controllers/modal';
+import { useDispatch } from 'react-redux';
 import { messages } from '../messages';
 
-const LABEL_WIDTH = 105;
+const PROJECT_NAME_FIELD = 'projectName';
+export const AddProjectModal = ({ data = {}, handleSubmit, invalid }) => {
+  const dispatch = useDispatch();
+  const { formatMessage } = useIntl();
+  const { onSubmit } = data;
 
-@withModal('addProjectModal')
-@injectIntl
-@reduxForm({
-  form: 'addProjectForm',
-  validate: ({ projectName }) => ({
-    projectName: bindMessageToValidator(validate.projectName, 'projectNameLengthHint')(projectName),
-  }),
-  asyncValidate: ({ projectName }) => validateAsync.projectNameUnique(projectName),
-  asyncChangeFields: ['projectName'],
-  asyncBlurFields: ['projectName'], // validate on blur in case of copy-paste value
-})
-export class AddProjectModal extends Component {
-  static propTypes = {
-    intl: PropTypes.object.isRequired,
-    data: PropTypes.object,
-    dirty: PropTypes.bool,
-    handleSubmit: PropTypes.func,
+  const onCreateProject = ({ projectName }) => {
+    onSubmit(projectName);
   };
+  const hideModal = () => dispatch(hideModalAction());
```

Comment:
Now this modal is located on the `instance` level, please move it to the corresponding page folder to the `organization` level.